src/rbd_fuse/rbd-fuse.cc

```diff
@@ -941,9 +946,13 @@ int main(int argc, const char *argv[])
 		exit(1);
 	}
 
-	for (auto& it : filter_args) {
-		if (it.second) {
+	for (auto& it : filter_options) {
+		if (it.second != OPT_NOT_FOUND) {
 			arg_vector.push_back(it.first.c_str());
+			if (require_arg_options.find(it.first) != require_arg_options.end() &&
```

Comment:
nit: you could use `require_arg_options.count(it.first)` to make it shorter.

Reply:
Thank you for your suggestion